vilebot/files/inspirationsindex.txt

```diff
@@ -103,7 +103,7 @@
 373
 376
 378
-379:$100 invested at 7% interest for 100 years will become $100,000, at
```

Comment:
What is actually causing the problem? Is it `%` anywhere, symbols generally or specifically `%`/symbol at the start of a sentence? Are the removals in this file just a precaution?